### PR TITLE
Challenge 1 in Python

### PR DESCRIPTION
**Pull request by slansford**

### CODE REVIEW

**Comment by wOstensen** [comment]
You can do the input function inside the print function, that way you'll have one single line.

**Comment by MJUIUC** [comment]
Yea this looks pretty solid. I didn't know you could write inputs that way so I learned something!

File: challenge_1/python/slansford/src/challenge_1.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#Program asks user for word, reverses word, and returns the reversed word
+
+#Asks user for input and reverses letters of word
+
+rev_word = input('What word would you like reversed? ')[::-1]
```
Comment (jarydremillard) [inline challenge_1/python/slansford/src/challenge_1.py]: Good use! Efficient! Its exactly what I wanted to see for this challenge.

File: challenge_1/python/slansford/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Prints reversed string provided by user input. Uses Python 3.6
```
Comment (jarydremillard) [inline challenge_1/python/slansford/README.md]: Thank you for including the version!

**Pull request merged**